stable/phpbb/values.yaml

```diff
@@ -32,15 +36,49 @@ phpbbEmail: user@example.com
 # smtpUser:
 # smtpPassword:
 
+##
+## External database configuration
+##
+externalDatabase:
+  ## Database host
+  # host:
```

Comment:
References to `.Values.externalDatabase.host` and other second level values will result in rendering errors. Uncommenting these values might be a clean way of dealing with these rendering issues.

Reply:
Done